website/docs/docs/build/sl-getting-started.md

```diff
@@ -26,14 +26,21 @@ This getting started page recommends a workflow to help you get started creating
 New to dbt or metrics? Try our [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template) to help you get started!
 :::
 
+## Install MetricFlow
+
+Before you begin, make sure you install the `metricflow` and [dbt adapter](/docs/supported-data-platforms) in the CLI using the `pip install "dbt-metricflow[your_adapter_name]"` command. This is because you're installing MetricFlow as an extension of the dbt adapter. Currently, the supported adapters are Snowflake and Postgres (BigQuery, Databricks, and Redshift coming soon).
```

Comment:
```suggestion
Before you begin, make sure you install the `metricflow` and [dbt adapter](/docs/supported-data-platforms) via PyPI in the CLI using the `pip install "dbt-metricflow[your_adapter_name]"` command. This is because you're installing MetricFlow as an extension of the dbt adapter. Currently, the supported adapters are Snowflake and Postgres (BigQuery, Databricks, and Redshift coming soon).
```